### PR TITLE
1.6 release team added

### PR DESCRIPTION
**Pull request by idvoretskyi**

### CODE REVIEW

File: release-1.6/release_team.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** |
```
Comment (spiffxp) [inline release-1.6/release_team.md]: what do you think about adding slack and github aliases to this?

File: release-1.6/release_team.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+|  ------ | ------ |
+|  Lead | Dan Gillespie |
+|  **Secondary** |  |
+|  Features | Ihor Dvoretskyi |
+|  Release Branch | Anthony Yeh |
+|  **Infra** | Jeff Grafton |
+|  Docs | Devin Donnelly |
+|  Bugs | Caleb Miles |
+|  **Testing** | Maru Newby |
```
Comment (spiffxp) [inline release-1.6/release_team.md]: is there a purpose/meaning behind bold vs. non-bold headings here?

Comment (idvoretskyi) [inline release-1.6/release_team.md]: @spiffxp I've been using this [spreadsheet](https://docs.google.com/spreadsheets/d/1OvBDDzVeqzktyGZSD4-eMfnJgJros-86rgHQrrYjP-s/edit#gid=0) as a source, created by @ethernetdan - no objections to modify the font.

File: release-1.6/release_team.md (hunk 1 above)
Comment (idvoretskyi) [inline release-1.6/release_team.md]: Good idea. They are also may be useful at the [spreadsheet](https://docs.google.com/spreadsheets/d/1OvBDDzVeqzktyGZSD4-eMfnJgJros-86rgHQrrYjP-s/edit#gid=0).

**Comment by spiffxp** [comment]
@idvoretskyi who's the spreadsheet shared with?  I must not be a member of the right google group

**Comment by idvoretskyi** [comment]
@spiffxp sorry, I've thought it's public, while it's for release team only.

Anyway, your suggestions make sense, let @ethernetdan share his thoughts here.

File: release-1.6/release_team.md (hunk 2 above)
Comment (ethernetdan) [inline release-1.6/release_team.md]: The bolding was originally to denote which roles are missing - currently only Secondary is unfilled.

Comment (idvoretskyi) [inline release-1.6/release_team.md]: @ethernetdan thank you, I will update the fonts.

**Pull request merged**